csunplugged/tests/topics/views/test_all_curriculum_integration_view.py

```diff
@@ -0,0 +1,76 @@
+from django.urls import reverse
+from model_mommy import mommy
+from tests.BaseTestWithDB import BaseTestWithDB
+from tests.topics import create_topics_test_data
+from topics.models import Topic
+
+
+class AllCurriculumIntegrationViewTest(BaseTestWithDB):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.language = "en"
+
+    def test_all_curriculum_integration_view_with_no_integrations(self):
+        url = reverse("topics:all_curriculum_integrations")
+        response = self.client.get(url)
+        self.assertEqual(200, response.status_code)
+        self.assertEqual(len(response.context["curriculum_integrations"]), 0)
+
+    def test_all_curriculum_integration_view_with_one_integration(self):
+        topic = mommy.make(Topic)
```

Comment:
I think it is confusing to switch between the two, if we can do it ourselves, we might as well.